README.md

```diff
@@ -183,7 +184,9 @@ It will save that information to `~/.actrc`, please refer to [Configuration](#co
       --privileged                       use privileged mode
   -p, --pull                             pull docker image(s) even if already present
   -q, --quiet                            disable logging of output from steps
+      --rebuild                          rebuild local action docker image(s) even if already present (default true)
   -r, --reuse                            reuse action containers to maintain state
+      --rm                               automatically remove containers just before exit
```

Comment:
Is this implemented?

Reply:
This was implemented in #714 but missed on adding to readme
 `--container-cap-add` missed in #716